server/controllers/stock/core.js

```diff
@@ -508,6 +510,40 @@ async function getStockConsumptionAverage(periodId, periodDate, monthAverageCons
     GROUP BY i.uuid, d.uuid;
   `;
 
+  // the value (w.quantity/w.days) is the CMM
+  // is mentionned here as quantity to fit into the expected values
+  // for returned values
+  const queryConsumptionByDays = `
+  SELECT
+    w.consumption AS counted_consumption,
+    w.quantity AS consumed_quantity,
+    ROUND(w.quantity/w.days) AS quantity,
+    w.depot_text, w.code, w.text, w.days,
+    BUID(w.depot_uuid) depot_uuid, BUID(w.inventory_uuid) uuid
+  FROM 
+  (
+    SELECT
+      z.depot_text, z.text, SUM(IF(z.date, 1, 0)) days, SUM(z.consumption) consumption, SUM(z.quantity) quantity,
+      z.depot_uuid, z.inventory_uuid, z.code
+    FROM (
+      SELECT 
+        count(*) consumption, SUM(sm.quantity) quantity,  sm.date,
+        i.uuid inventory_uuid, i.text, i.code,
+        d.uuid depot_uuid, d.text AS depot_text
+      FROM stock_movement sm 
+      JOIN lot l ON l.uuid = sm.lot_uuid
+      JOIN inventory i ON i.uuid = l.inventory_uuid
+      JOIN depot d ON d.uuid = sm.depot_uuid
```

Comment:
There is a group by d.uuid,... which is useful for the `COUNT(*) consumption`

Reply:
But you can group by `depot_uuid`, right?  Isn't that the same thing?

It doesn't look like you need to JOIN on the depot table to group on the uuid, since it is already in the `stock_movement` table.